src/Components/ClusterRules/ClusterRules.js

```diff
@@ -369,7 +369,8 @@ const ClusterRules = ({ reports }) => {
         expandAll={{ isAllExpanded, onClick: collapseAll }}
         filterConfig={{
           items: filterConfigItems,
-          isDisabled: filteredRows.length === 0,
+          isDisabled:
+            reports.length === 0 ? filteredRows.length === 0 : undefined,
```

Comment:
Check for the length of the `filteredRows` is unnecessary: there is no case when we want to disable the toolbar depending on the filtered entities (and this is a thing I confused when I was implementing this piece of code). It needs to depend only on `reports` - the initial array of entities before we filter them. Thus, I suggest the following:
```suggestion
            reports.length === 0;

Reply:
Yep this is a better variant :) thanks for the comment